overlord/servicestate/quota_handlers_test.go

```diff
@@ -1260,6 +1267,253 @@ func (s *quotaHandlersSuite) TestQuotaUpdateChangeMemLimit(c *C) {
 	c.Assert(err, ErrorMatches, "cannot update limits for group \"foo\": cannot decrease memory limit, remove and re-create it to decrease the limit")
 }
 
+func (s *quotaHandlersSuite) TestCreateJournalQuota(c *C) {
+	r := s.mockSystemctlCalls(c, join(
+		// CreateQuota for foo
+		systemctlCallsForCreateQuota("foo", "test-snap"),
+	))
+	defer r()
+
+	// Add fake handlers for the setup-profiles task which should be invoked
+	// when creating the journal quota.
+	var setupProfilesCalled int
+	fakeHandler := func(task *state.Task, _ *tomb.Tomb) error {
+		task.State().Lock()
+		_, err := snapstate.TaskSnapSetup(task)
```

Comment:
shouldn't we check that the right/expected snap and revision is mentioned in the result of this, same for the other tests here?